src/main/java/me/magnet/consultant/ConfigUpdater.java

```diff
@@ -164,4 +183,16 @@ private Properties updateConfig(List<KeyValueEntry> entries) {
 		return properties;
 	}
 
+	public void shutDown() {
+		shutdownBegun.set(true);
+		if (request != null) {
+			try {
+				request.abort();
+			}
+			catch (RuntimeException e) {
+				log.error("Could not abort request. No biggy, just FYI.", e);
```

Comment:
Maybe to go all bruh in log statements :). "Could not abort request while shutting down." And if it's "no biggy" it's not "error" either.